app/meta_development.go

```diff
@@ -0,0 +1,40 @@
+//go:build !no_metadata
```

Comment:
Instead of more complexity (more files) in the `app/` folder, can you not use the new `internal/build` package and add a `build.NoMetadata` constant?

Reply:
You are right that it won't result in less files but it would avoid adding more files to the already crowded app/ folder. The files in the other package are easier to grasp what they do. It is also consistent with how we check other build tags using that internal/build package.

Reply:
OK done.

Just to be clear this change means:
* 1 extra file
* A package public constant checked in 1 place
* the toml package cannot be ignored during compile

Reply:
The first two are correct but will have less meaning if we use them in more places in the future.

The third one, about toml not being ignored should not be correct. As the NoMetadata values is constant, the compiler can dead-code eliminate the branches where it is used.

Reply:
> The third one, about toml not being ignored should not be correct. As the NoMetadata and Type values are constant, the compiler can dead-code eliminate the branches where they are used.

I guess this is getting a little into semantics but I think we are both right. The package cannot be ignored during compile - however it may be possible that the compiler later optimises it out as you say.

Reply:
Yup. It will likely be downloaded but it should not be part of the binary :)